### PR TITLE
Adding translations of articles about grippers and assembling a FPV set

### PR DESCRIPTION
**Pull request by Alamoris**

### CODE REVIEW

File: docs/en/fpv_clover_4_2.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# Installing and configuring FPV equipment
```
Comment (okalachev) [inline docs/en/fpv_clover_4_2.md:1]: File is not added to the summary.

**Review by okalachev — CHANGES_REQUESTED** [review]

File: docs/en/magnetic_grip.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# Assembling and setting up the electromagnetic gripper
```
Comment (okalachev) [inline docs/en/magnetic_grip.md:1]: File is not added to the summary.

**Review by okalachev — CHANGES_REQUESTED** [review]

File: docs/en/magnetic_grip.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+<img src="../assets/magnet_grip/magnet_raspberry.png" width=300 class="zoom border center">
+
+An example of the code activating the magnetic gripper can be found [here](gpio.md#connecting-an-electromagnet).
+
+## Connecting to Arduino
+
+Connect the gripper to the Arduino Nano board in order to control it manually.
+
+It is convenient to place it on the same soldering board -- insert it into the appropriate holes and solder it from the back to the board.
```
Comment (okalachev) [inline docs/en/magnetic_grip.md:52]: Use simple `–`.

File: docs/en/magnetic_grip.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+4. Connect the center leg of the transistor and the leg of the Schottky diode (opposite to the gray marking strip).
+
+    <img src="../assets/magnet_grip/magnet3.png" width=300 class="zoom border center">
+
+5. Cut the required amount of magnetic grab wire and solder it to the pins of the Schottky diode.
+
+    <img src="../assets/magnet_grip/magnet4.png" width=300 class="zoom border center">
+
+6. Solder the *Dupont* -dad wires to the transistor and diode leg (red, black wires), and the *Dupont* -mama wire to the opposite transistor leg (white wire).
```
Comment (okalachev) [inline docs/en/magnetic_grip.md:28]: Dupont "dad" and "mama"? WTF, @Alamoris.

File: docs/en/mechanical_grip.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# Assembling and setting up a mechanical gripper
```
Comment (okalachev) [inline docs/en/mechanical_grip.md:1]: File is not added to the summary.

**Review by okalachev — CHANGES_REQUESTED** [review]

File: docs/en/magnetic_grip.md (hunk 4 above)
Comment (Alamoris) [reply docs/en/magnetic_grip.md]: Uuups, fix

File: docs/en/mechanical_grip.md (hunk 5 above)
Comment (Alamoris) [reply docs/en/mechanical_grip.md:1]: Add

File: docs/en/magnetic_grip.md (hunk 3 above)
Comment (Alamoris) [reply docs/en/magnetic_grip.md]: Add "—" symbol

File: docs/en/magnetic_grip.md (hunk 2 above)
Comment (Alamoris) [reply docs/en/magnetic_grip.md:1]: Add

File: docs/en/fpv_clover_4_2.md (hunk 1 above)
Comment (Alamoris) [reply docs/en/fpv_clover_4_2.md:1]: Add

File: docs/en/SUMMARY.md [hunk 6]
```diff
@@ -85,7 +88,7 @@
   * [LED strip (legacy)](leds_old.md)
   * [Contribution Guidelines](contributing.md)
   * [Migration to v0.20](migrate20.md)
-* [Clover-based projects](projects.md)
+  * [Clover-based projects](projects.md)
```
Comment (okalachev) [inline docs/en/SUMMARY.md:91]: What is this?

**Review by okalachev — CHANGES_REQUESTED** [review]

File: docs/en/SUMMARY.md (hunk 6 above)
Comment (Alamoris) [reply docs/en/SUMMARY.md]: Fix

**Pull request merged**